CHANGELOG.md

```diff
@@ -7,11 +7,34 @@ This projects adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 Up to version 8.0.0, the `major.minor` version numbers were based on the PHP version for which compatibility check support was added, with `patch` version numbers being specific to this library.
 From version 8.0.0 onwards, [Semantic Versioning](http://semver.org/) is used.
 
+**IMPORTANT**: The 8.0.0 release contains a **breaking change**. Please read the below information carefully before upgrading!
+
 
 ## [Unreleased]
 
 _Nothing yet._
 
+## [8.0.1] - 2017-08-07
+
+See all related issues and PRs in the [8.0.1 milestone].
+
+### Added
+- PHP 7.2 trailing comma in group use statement is now allowed
+- :fire: PHP 7.2 deprecated functions
+- :fire: PHP 7.2 deprecated/removed ini directives
+- :sparkles: PHP 7.2 (unset) cast handling
```

Comment:
:star2: New `DeprecatedTypeCasts` sniff to detect deprecated and removed type casts, such as the `(unset)` type cast as deprecated in PHP 7.2

- As this is a new sniff, it should go to the top of the list
- A second entry is needed for the second new sniff:
    :star2: New `NewTypeCasts` sniff to detect type casts not present in older PHP versions such as the `(binary)` type cast as added in PHP 5.2.1.